addins/Cake.Recipe.yml

```diff
@@ -0,0 +1,11 @@
+Name: Cake.Recipe
```

Comment:
Should be filtered as it is not an addin

Reply:
I should probably exclude all packages containing "recipes" from the yaml generation process

Reply:
Done. Packages containing "recipes" will be excluded from YAML generation